requirements/test-requirements.txt

```diff
@@ -6,6 +6,7 @@ pytest==8.1.0
 pytest-cov
 pytest-timeout
 pytest-localserver==0.5.0
+pytest-asyncio==0.23.8
```

Comment:
Do we need to pin asyncio to this version?

Reply:
Removed this line, as we pytest-asyncio not needed anymore.